framework/src/base/FEProblemBase.C

```diff
@@ -4428,12 +4431,14 @@ FEProblemBase::adaptMesh()
     else
     {
       _console << "Mesh unchanged, skipping remaining steps..." << std::endl;
```

Comment:
Wait, the `std::endl` is definitely needed because the `break` right below that breaks you out of the for loop which skips the `std::flush`, meaning the next line to get printed to the console gets stuck on the end of this one.

I had put the `std::flush` in there originally because I wasn't seeing mesh adaptivity progress after each step, but rather all of it at once after adaptivity was done, which I didn't like. However, that doesn't appear to be the case anymore. Did something change?